crates/cext/src/pointers.rs

```diff
@@ -0,0 +1,48 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2024
```

Comment:
```suggestion
// (C) Copyright IBM 2025
```

Reply:
The contents of this file were just moved from the original unchanged and they had a 2024 copyright header so I just preserved that.